src/main/kotlin/ar/edu/unahur/obj2/vendedores/Vendedor.kt

```diff
@@ -35,17 +40,58 @@ class VendedorFijo(val ciudadOrigen: Ciudad) : Vendedor() {
   override fun puedeTrabajarEn(ciudad: Ciudad): Boolean {
     return ciudad == ciudadOrigen
   }
+  // Esta función devuelve un booleano (false)
+  fun esInfluyente() = false
```

Comment:
Ok. Faltó definirlo como método abstracto en `Vendedor`, ya que si no no se pueden usar los vendedores polimórficamente. Es decir que algo como `listOf(VendedorFijo(...), Viajante(...)).filter { it.esInfluyente() }` no compilaría